pages/_slug/index.vue

```diff
@@ -37,8 +26,12 @@ export default {
 			]
 		};
 	},
-	fetch ({ store }) {
-		return store.dispatch("fetchSitePages");
+	asyncData ({ $axios, $payloadURL, route, store }) {
+		if (process.static && process.client && $payloadURL) {
+			return $axios.$get($payloadURL(route));
+		}
+
+		return store.state.sitePages.find(oneSitePage => oneSitePage.slug === route.params.slug);
```

Comment:
Shall  `await store.dispatch("fetchSitePages");` be added before this line?

Reply:
No need because it is called in `actions.js` within `nuxtServerInit`